hera_qm/xrfi.py

```diff
@@ -222,6 +224,223 @@ def robust_divide(num, den):
     return out
 
 
+# ############################################################################
+# Functions for flagging auto-correlations
+# ############################################################################
+
+def dpss_flagger(data, freqs, filter_centers, filter_half_widths, wgts=None,
+                 nsig=10, mode="dpss_solve", noise=None, eigenval_cutoff=[1e-9],
+                 suppression_factors=[1e-9], cache={}):
+    """
+    Flag visibilities by filtering data with discrete prolate spheroidal sequences
+
+    Parameters:
+    ----------
+    data: np.ndarray
+        2D data array of the shape (time, frequency)
+    freqs: np.ndarray
+        1D array of frequencies present in the data
+    filter_centers: array-like
+        list of floats of centers of delay filter windows in nanosec
+    filter_half_widths: array-like
+        list of floats of half-widths of delay filter windows in nanosec
+    wgts: np.ndarray
+        2D np.ndarray of real weights. Must be the same shape as data.
+    nsig: float, default=10
```

Comment:
A default of 10 is different from a lot of the other defaults in this module (usual 5 or 6). Is there a reason for this number?

Reply:
This is just what Aaron set in the fastcal notebook for the channel difference flagger. I just changed the default value to `nsig=6` and it didn't significantly change the result on the test data. I'll go ahead and set it to 6